lib/fiddle/jruby.rb

```diff
@@ -158,6 +158,17 @@ def call(*args, &block);
           end
```

Comment:
I think it would be good to rename this file to `ffi_backend.rb` or so and avoid a JRuby namespace since it is effectively independent from JRuby but it doesn't like that currently due to file naming and things like `Fiddle::JRuby`.
It's helpful to have a small diff at this point though, so I think that's best done later, e.g. a separate PR.

Reply:
I think that we don't need to do it because this is a short-term workaround for TruffleRuby.
This file will be used by only JRuby eventually.

Reply:
Since the FFI backend seems to work well I think we should just use it long term, for both JRuby and TruffleRuby.